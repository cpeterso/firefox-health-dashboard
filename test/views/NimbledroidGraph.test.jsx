import React from 'react';
import renderer from 'react-test-renderer';
import NimbledroidGraph from '../../src/views/NimbledroidGraph';

it('renders correctly', () => {
  const props = {
    site: 'test site',
    location: {
      search: 'your search',
    },
  };
  const tree = renderer
    .create(<NimbledroidGraph location={props.location} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
