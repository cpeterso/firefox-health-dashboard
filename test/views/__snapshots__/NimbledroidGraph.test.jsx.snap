// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="DashboardPage-root-1"
>
  <div
    className="DashboardPage-title-2"
  >
    <h1
      style={
        Object {
          "display": "inline",
        }
      }
    >
      Android
      <small
        className="DashboardPage-subtitle-3"
      >
        Release criteria
      </small>
    </h1>
    <span
      style={
        Object {
          "float": "right",
        }
      }
    >
      <a
        href="https://github.com/mozilla-frontend-infra/firefox-health-dashboard/"
      >
        <img
          alt="Go to Github Repo"
          src="test-file-stub"
          style={
            Object {
              "padding": "0 0.5rem 0 0",
              "width": "2rem",
            }
          }
        />
      </a>
    </span>
  </div>
  <div>
    <div
      className="Section-header-5"
    >
      <h2
        className="Section-title-6"
      >
        <span>
          Nimbledroid
          <span
            className="Section-subtitle-7"
          >
            GeckoView Example vs Chrome Beta
          </span>
        </span>
      </h2>
    </div>
    <div
      className="Section-content-4"
    />
  </div>
</div>
`;
